lib/api/config.rb

```diff
@@ -0,0 +1,6 @@
+require "config"
+
+module Api
+  CONFIG_PATH = File.expand_path(File.join("..", "..", "..", "config", "api.yml"), __FILE__).freeze
+  Config = ::Config::Options.new(YAML.load_file(CONFIG_PATH))
```

Comment:
See my other comments where I gave a better alternative that is slightly differerent.